.ci/azure-pipelines/build_linux.bash

```diff
@@ -7,20 +7,34 @@
 #
 # Below is a list of configuration variables used from environment.
 #
+# Predefined variables:
+#
 #  BUILD_BINARIESDIRECTORY - The local path on the agent that can be used
 #                            as an output folder for compiled binaries.
 #  BUILD_SOURCESDIRECTORY  - The local path on the agent where the
 #                            repository is downloaded.
 #
+# Defined on Azure Pipelines:
+#
+#  BUILD_NUMBER_TOKEN      - Access token for the build number page on our server.
+#
 
-VER=$(python scripts/mumble-version.py)
+if [[ -n "$BUILD_NUMBER_TOKEN" ]]; then
+	VERSION=$(python "scripts/mumble-version.py" --project)
+	BUILD_NUMBER=$(curl "https://mumble.info/get-build-number?version=$VERSION&token=$BUILD_NUMBER_TOKEN")
+else
+	BUILD_NUMBER=0
+fi
```

Comment:
Actually I think we should not default to `0` in order to let cmake error about undefined build number (as `packaging=ON`). Otherwise we could accidentally always package with build number 0 which is actually against the whole point of its introduction ^^

Reply:
It works as you described.

Reply:
Oh right :facepalm: 
I guess I have stared at this PR for too long now :sweat_smile: